src/ORM/README.md

```diff
@@ -0,0 +1,112 @@
+# CakePHP ORM
+
+The CakePHP ORM provides a powerful and flexible way to work with relational
+databases. Using a datamapper pattern the ORM allows you to manipulate data as
+entities allowing you to create expressive domain layers in your applications.
+
+## Connecting to the Database
+
+The CakePHP ORM is compatible with:
+
+* MySQL 5.1+
```

Comment:
Just in general, do you really want to "officially" support (e.g. warrant) of such an old version of MySQL? Why not 5.5+? ... that's from 2010?